calls.go

```diff
@@ -167,7 +169,7 @@ func (c *CallService) GetCallsInRange(start time.Time, end time.Time, data url.V
 	d.Del("Page") // just in case
 	if start != Epoch {
 		startFormat := start.UTC().Format(APISearchLayout)
-		d.Set("StartTime>", startFormat)
+		d.Set("StartTime>=", startFormat)
```

Comment:
Right but the changes you made wouldn't match the API documentation. The way it is now matches the API documentation. They would do instead:

```
$ curl -G https://api.twilio.com/2010-04-01/Accounts/AC6bc21af903cc765a9d7f7e0467ec812a/Calls.json \
    -d "Status=completed" \
    -d "StartTime%3E%3D=2009-07-06" \
    -u 'AC6bc21af903cc765a9d7f7e0467ec812a:your_auth_token'
```